jac-uk/apply
# #961 Enable candidates to apply for a vacancy in Welsh

## What's included?
Enable candidates to apply for a vacancy in Welsh.

Development Plan:
- [x] Indicate application was made in Welsh (_language: 'cym') when applying for an application and submitting it.
- [x] Go through all the pages during the application process and check which HTML textNode needs to translate into Welsh.
- [x] Write a listener triggered when HTML DOM elements update.
- [x] Write a function to update all textNodes in Welsh except:
    - [x] input, textarea
    - [x] Answer in review (need apply a class to mark it not to translate)
    - [x] Input type=file
- [x] Implement a way to switch back to English.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Find a vacancy with Welsh posts and click the "Apply in Welsh" button.
2. Check if the text is translated correctly into Welsh.
3. Check if you are able to switch the languages between English and Welsh.
4. Back to the home page check if the language is switched back to English.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/208420580-e90ca4f9-5385-477d-a6ca-53aa2433791a.mov

Screenshots:

<img width="1142" alt="Screenshot 2022-12-16 at 13 00 55" src="https://user-images.githubusercontent.com/79906532/208104160-ad861294-39e5-42fd-8820-d2ce4a14a3b6.png">

<img width="992" alt="Screenshot 2022-12-19 at 11 45 18" src="https://user-images.githubusercontent.com/79906532/208418894-69d3db5a-75be-46d1-9dd3-295aeb77d165.png">

---
PREVIEW:DEVELOP
